pgtrigger/core.py

```diff
@@ -823,12 +823,15 @@ def get(*uris):
         return list(registry.values())
 
 
-def install(*uris):
+def install(*uris, **connection_kwargs):
     """
     Install registered triggers matching URIs or all triggers if URIs aren't
     provided. If URIs aren't provided, prune any orphaned triggers from the
     database
     """
+    if 'using' in connection_kwargs:
+        global connection
+        connection = connections[connection_kwargs['using']]
```

Comment:
I would really prefer not to have this global hack in here. This will override the connection for all subsequent functions in the same loaded process (like the `uninstall` function). This is especially problematic if you use the `ignore` functionality to ignore trigger execution dynamically for certain operations in your code.

I will post a follow-up comment on this PR with some more thoughts about the can of worms that multiple databases exposes for pgtrigger. I believe that there is a solution for multiple databases, but it might be a bit more manual